tools/wtdbg/wtdbg_kbm.xml

```diff
@@ -0,0 +1,82 @@
+<tool id="wtdbg_kbm" name="wtdbg_kbm" version="1.2.8.1">
+   <description>- kbm is a simple instance which implemented kmer-binmap
+         it maps query sequence against reference by kmer matching</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="requirements" />
+    <command detect_errors="exit_code"><![CDATA[
+        kbm 
+        -t \${GALAXY_SLOTS:-4}
+        -d '$input1' 
+        -i '$input2' 
+        -o reads.kbmap
+    ]]></command>
+    <inputs>
+        <param type="data" name="input1" format="fasta" />
+        <param type="data" name="input2" format="fasta" />
+    </inputs>
+    <outputs>
+        <data name="output1" format="kbmap" from_work_dir="reads.kbmap" />
```

Comment:
I'm not sure what file format this is, but Galaxy does not know it.
You could use tabular if you like.